jerry-core/ecma/base/ecma-extended-info.c

```diff
@@ -13,10 +13,12 @@
  * limitations under the License.
  */
 
-#include "byte-code.h"
-#include "ecma-helpers.h"
 #include "ecma-extended-info.h"
 
+#include "ecma-helpers.h"
+
```

Comment:
These newlines are unnecessary.

Reply:
That's valid.
Include order separated to incluce blocks:

1. own header
2. system header
3. jerry* header
4. ecma* header
5. everything else.